dev-server/ui-playground/shared/components/PlaygroundApp.js

```diff
@@ -14,29 +21,37 @@ function HomeRoute() {
   );
 }
 
-const routes = [
+const routes = /** @type PlaygroundRoute[] */ ([
   {
     route: /^\/?$/,
     title: 'Home',
     component: HomeRoute,
   },
-  {
-    route: '/menu',
-    title: 'Menu',
-    component: MenuPatterns,
-  },
   {
     route: '/shared-buttons',
-    title: 'Buttons (Shared)',
+    title: 'Buttons',
     component: SharedButtonPatterns,
   },
-];
+]);
 
 const demoRoutes = routes.filter(r => r.component !== HomeRoute);
 
-export default function PlaygroundApp() {
-  const baseUrl = '/ui-playground';
-  const [route, navigate] = useRoute(baseUrl, routes);
+/**
+ * @typedef PlaygroundAppProps
+ * @prop {string} [baseURL]
+ * @prop {PlaygroundRoute[]} [extraRoutes] - Routes to add to the common routes
+ */
+
+/**
+ *
+ * @param {PlaygroundAppProps} props
+ * @returns
```

Comment:
There is an empty / redundant `@returns` tag here.

Reply:
Bless my editor and its over-helpfulness sometimes.